### ISSUE TITLE
Suggestion about utility belt

### ISSUE THREAD

**Issue by CreyZHere** [Lizard-Of-Oz/Inventorio#252]
Make it so that dropped stackable items go directly to the utility belt when picked up if there's already the same item in the belt and the stack isnt full (f.e if you have some torches in the utility belt and you place one down and then break it, make it so that it gets picked up in the slot of the belt with the preexisting non-full stack of torches)

**Comment by Lizard-Of-Oz** [comment]
It's supposed to already work like that. Are you sure it's not working for you?

**Comment by CreyZHere** [comment]
> It's supposed to already work like that. Are you sure it's not working for you?

Yes, pretty sure. Tried it already 50+ times, each and every time i had to manually pick up the torch in my inventory and then move it to the corresponding stack in the utility belt. Playing on 1.20.1 with the latest version of inventorio.

Also idk if i should create another issue for this or not, but do you know by any chance why when items get picked up in the extra inventory from deep pockets enchantment, they stack up to 3-4 for every slot and even if there are same items on the ground they dont get picked up?

**Comment by Lizard-Of-Oz** [comment]
Do you have any other mods that interact with the inventory? Sorting mods, pickup-related mods, etc.

**Comment by Lizard-Of-Oz** [comment]
Actually, nevermind - this indeed broke in 1.20

**Comment by Lizard-Of-Oz** [comment]
Thanks for the bug report!

**Comment by CreyZHere** [comment]
> Actually, nevermind - this indeed broke in 1.20

Alright then, thanks for the fast replies and thanks for letting me know
